### PR TITLE
PSAP-444: NTO - using built-in TuneD profiles

### PR DESCRIPTION
**Pull request by jmencak**
Changes:
  - Add documentation wrt using TuneD profiles shipped within the TuneD daemon.
  - Make the custom tuning using the openshift-control-plane TuneD profile look nicer by changing the formatting to yaml.

### CODE REVIEW

**Comment by jmencak** [comment]
/cc @ahardin-rh 
/cc @skordas 
Simon, could you please provide a QE review?  Thank you!

**Comment by jmencak** [comment]
Note I'm only mentioning the issue https://bugzilla.redhat.com/show_bug.cgi?id=1825882 in the comments as it is planned to be addressed for 4.8. I've already proposed a patch to fix the issue.

File: modules/custom-tuning-example.adoc [hunk 1]
```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
...
+
+*Using built-in TuneD profiles*
+
+Given the successful rollout of the NTO-managed DaemonSet,
+the TuneD operands all manage the same version of the
+link:https://github.com/redhat-performance/tuned/[TuneD daemon].  To list
+the built-in TuneD profiles supported by the daemon, query any TuneD pod
+in the following way:
+
+[source,yaml]
```
Comment (skordas) [inline modules/custom-tuning-example.adoc:54]: It looks like a terminal command ;)
[source,terminal]

File: modules/custom-tuning-example.adoc [hunk 2]
```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
 daemon profiles shipped within the default Tuned CR. The example above uses the
 default `openshift-control-plane` profile to accomplish this.
 ====
+
+*Using built-in TuneD profiles*
+
+Given the successful rollout of the NTO-managed DaemonSet,
```
Comment (ahardin-rh) [inline modules/custom-tuning-example.adoc:48]: s/DaemonSetdaemon set per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references

File: modules/custom-tuning-example.adoc [hunk 3]
```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
...
+the TuneD operands all manage the same version of the
+link:https://github.com/redhat-performance/tuned/[TuneD daemon].  To list
```
Comment (ahardin-rh) [inline modules/custom-tuning-example.adoc:50]: We can't link to GitHub pages unless we get special permission from Support
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

File: modules/custom-tuning-example.adoc [hunk 4]
```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
...
+    name: openshift-node-hpc-compute
+
+  recommend:
+  - match:
+    - label: tuned.openshift.io/openshift-node-hpc-compute
+    priority: 20
+    profile: openshift-node-hpc-compute
+----
+
+Note the example above includes the default `openshift-node` TuneD daemon profiles shipped within the default Tuned CR. This is to take advantage of the OpenShift specific tuning for compute nodes.
```
Comment (ahardin-rh) [inline modules/custom-tuning-example.adoc:84]: s/Note the example/The example

s/This is to take advantage of the OpenShift specific/The default profiles use OpenShift-specific tuning

**Review by ahardin-rh — COMMENTED** [review]
Just a few minor comments from me. Thanks! 

Is this work tied to a JIRA card or bug for tracking?

**Comment by ashishkamra** [comment]
@ahardin-rh https://issues.redhat.com/browse/PSAP-444
cc @jeniferh if you want to provide any feedback (your questions triggered this effort :))

**Comment by jeniferh** [comment]
Looks good. Thanks, this makes it clear how to use built-in profiles.

File: modules/custom-tuning-example.adoc (hunk 4 above)
Comment (jmencak) [reply modules/custom-tuning-example.adoc:84]: The second change would imply that the default profiles provide OpenShift-specific tuning to compute nodes only. This is not true. The default profiles also provides tuning for the control plane. Would changing the entire sentence to:

```
In addition to the built-in `hpc-compute` profile, the example above includes
the `openshift-node` TuneD daemon profile shipped within the default
Tuned CR to use OpenShift-specific tuning for compute nodes.
```

help?

**Comment by jmencak** [comment]
Many thanks for all the reviews! Very helpful as always. Hopefully, all should be addressed now.

**Review by skordas — APPROVED** [review]
/lgtm

File: modules/custom-tuning-example.adoc (hunk 4 above)
Comment (ahardin-rh) [reply modules/custom-tuning-example.adoc]: @jmencak Your updates look good to me. Thank you!

**Pull request merged**